api-jackson/src/main/java/com/basistech/util/jackson/EnumModule.java

```diff
@@ -37,6 +37,7 @@ public EnumModule() {
     }
 
     public void setupModule(SetupContext context) {
+        super.setupModule(context);
```

Comment:
Release note?  I know there's no real external change yet... but might help when trying
to figure out why we needed a new version of common-api.